docs/docsite/rst/dev_guide/developing_collections_structure.rst

```diff
@@ -168,7 +168,7 @@ roles directory
 
 Collection roles are mostly the same as existing roles, but with a couple of limitations:
 
- - Role names are now limited to contain only lowercase alphanumeric characters, plus ``_`` and start with an alpha character.
+ - Role names can only contain lowercase alphanumeric characters and underscores ( ``_`` ). Role names prefixed with an underscore are not intended for direct user invocation and should be reserved for internal collection roles only.
```

Comment:
```suggestion
 - Role names can contain only lowercase alphanumeric characters and underscore ( ``_`` ) characters. Role names prefixed with an underscore are not intended for direct user invocation and should be reserved for internal collection roles.
```

Reply:
There are also spaces around the underscore in the brackets which should be removed.